lib/client.js

```diff
@@ -132,7 +132,7 @@ class CloudantClient {
         var creds = self._cfg.creds || {};
         self._plugins.push(
           // instantiate plugin
-          new Plugin(self._client, Object.assign({ serverUrl: creds.outUrl }, cfg))
+          new Plugin(self._client, Object.assign({ serverUrl: creds.outUrl }, cfg), self.requestDefaults)
```

Comment:
Do we really need to change this constructor? IIUC `self._client` is a `request` object already and we already pass that so we can just call `.defaults()` on that object in the plugins.